modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/app/App.js

```diff
@@ -1070,7 +1087,10 @@ class App extends EventEmitter {
 
 			return;
 		}
-		this.maybeNavigate_(event.delegateTarget.href, event);
+		this.maybeNavigate_(
+			event.target.closest(this.getLinkSelector()).href,
```

Comment:
This is slightly annoying and feels like it could be handled in our `delegate(container, selector, callback)` callback where `selector` is already known.

/cc @bryceosterhaus, @kresimir-coko, @carloslancha, @wincent